### PR TITLE
testing/fceux: new aport

### PR DESCRIPTION
**Pull request by Danct12**
FCEUX is a Nintendo Entertainment System (NES), Famicom, Famicom Disk System (FDS), and Dendy emulator.

Signed-off-by: Danct12 <danct12@disroot.org>

### CODE REVIEW

File: testing/fceux/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Contributor: Danct12 <danct12@disroot.org>
+# Maintainer: Danct12 <danct12@disroot.org>
+pkgname=fceux
+pkgver=2.2.3
+pkgrel=0
+pkgdesc="Fast and ultra-compatible NES/Famicom emulator"
+url="http://www.fceux.com"
+arch="all"
+license="GPL"
```
Comment (maxice8) [inline testing/fceux/APKBUILD]: needs SPDX identifier

File: testing/fceux/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+depends="sdl gd gtk+2.0"
```
Comment (maxice8) [inline testing/fceux/APKBUILD]: there should be automatically found by the package manager by looking at the linked libraries

File: testing/fceux/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+depends_dev="sdl-dev gd-dev gtk+2.0-dev scons mesa-dev glu-dev"
```
Comment (maxice8) [inline testing/fceux/APKBUILD]: I think scons isn't meant to be in the developer dependencies but rather in makedepends

File: testing/fceux/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+makedepends="$depends_dev"
+install=""
```
Comment (maxice8) [inline testing/fceux/APKBUILD]: empty variable can be removed for brevity

File: testing/fceux/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+subpackages="$pkgname-doc"
+source="https://downloads.sourceforge.net/project/fceultra/Source%20Code/$pkgver%20src/$pkgname-$pkgver.src.tar.gz
+	scons-python3.patch"
+options="!check" # no testsuite
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (maxice8) [inline testing/fceux/APKBUILD]: default value can be removed

File: testing/fceux/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+prepare() {
```
Comment (maxice8) [inline testing/fceux/APKBUILD]: Why define prepare if you're going to define it as the default definition ?

remove this

File: testing/fceux/APKBUILD [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+        default_prepare
+}
+
+build() {
+	cd "$builddir"
```
Comment (maxice8) [inline testing/fceux/APKBUILD]: no need for cd "$builddir" statements, it is done implicitly

File: testing/fceux/APKBUILD [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+	scons -C "$pkgname-$pkgver"
+}
+
+package() {
+	cd "$builddir"
```
Comment (maxice8) [inline testing/fceux/APKBUILD]: same here

File: testing/fceux/APKBUILD (hunk 2 above)
Comment (Danct12) [reply testing/fceux/APKBUILD]: So.. this depends line could be removed entirely?

**Pull request closed without merging**